First 2 sections

Timowest12 opened the pull request:
Created the first part of my portfolio, I accidentally worked ahead a little bit, i changed the color to slightly darker because i personally liked it more (I hope that is not against the assignment) I tried to construct my code as correct as possible, Yet sometimes i did not know if something was good practice or just tricks that i know.

DanteAlonsoHT commented on index.html:
```diff
@@ -0,0 +1,94 @@
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
```
Your design is nice, but I feel you can add, for instance, the right background color according to the Figma design you selected, we are going to check that you make almost an exact copy from the Figma design due to a lot of companies expect this ability from front-end developers.
- [ ] Try to add and fix more/some styles to make it more similar to the Figma design.
- [ ] Fix the menu logo size, make it smaller.
- [ ] [OPTIONAL] Add a little bit more padding for your body or main content to center a little bit your content 😉 

Figma design you selected  | Your design

![image](https://user-images.githubusercontent.com/60164239/133576893-4903b526-108a-4b93-8e83-1d8bd049a6b8.png)

DanteAlonsoHT commented on index.html:
```diff
@@ -0,0 +1,94 @@
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <section>
+      <nav>
+        <ul>
+          <li class="navName">Portfolio</li>
+          <li class="navIcon"><i class="fas fa-bars"></i></li>
+        </ul>
+      </nav>
+    </section>
```
- [ ] Instead of using `section` to contain your `nav`, you should use `header` to apply semantic HTML. For example:
```
<header>
  <nav>
     ...
  </nav>
</header>
```

DanteAlonsoHT commented on index.html:
```diff
@@ -0,0 +1,94 @@
+        <div class="lang">
+          <ul>
+            <li class="listhead">Language</li>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </div>
+        <div class="frameworks">
+          <ul>
+            <li class="listhead">Frameworks</li>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="skill">
+          <ul class="skills">
+            <li class="listhead">Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </section>
+      <hr />
+      <section>
+        <div class="sideGradientright"></div>
+        <ul class="socials">
+          <li><i class="fab fa-github"></i></li>
+          <li><i class="fab fa-linkedin"></i></li>
+          <li><i class="fas fa-hand-peace"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li><i class="fab fa-facebook-square"></i></li>
+        </ul>
+      </section>
```
- [ ] All of these lines of code should be one section, only for your headline, please add all of this content inside a `section`.

DanteAlonsoHT commented on index.html:
```diff
@@ -0,0 +1,94 @@
+          <li><i class="fas fa-hand-peace"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li><i class="fab fa-facebook-square"></i></li>
+        </ul>
+      </section>
+    </main>
```
- [ ] The button to go to the "work" section is missing, please add a `button`/`a`.

Figma design | Your design

![image](https://user-images.githubusercontent.com/60164239/133578384-bf8827be-09b3-4722-9da0-bcdbceb2f8a6.png)

DanteAlonsoHT commented on styles/style.css:
```diff
@@ -0,0 +1,126 @@
+
+.listhead {
+  color: rgb(126, 126, 241);
+  margin-bottom: 20px !important;
```
- [ ] Please, remove`!important` value due to is a bad practice in CSS code. 😉

DanteAlonsoHT commented on README.md:
```diff
@@ -1,59 +1,39 @@
 
-> One paragraph statement about the project.
+> Mobile portfolio page
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+This is my Portfolio
```
- [ ] For this and the next projects you should add a better description, just try to explain what you did, 
what is the main scope, what kind of technologies or techniques you used, and so on. (Add like 3 lines or more at least in your description 😉).

DanteAlonsoHT reviewed (changes requested):
<h1> Status: Required Changes 🚫 </h1>

Hi @Timowest12, 

Amazing job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Do the best to replicate the Figma design you selected for this and the next protects about your portfolio😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Timowest12 commented:
Hi, I'm finished. I updated all the requested changes and made sure they looked very close to the original design. I also removed the bad practices in my CSS.

SajjadAhmad14 commented on index.html:
```diff
@@ -0,0 +1,107 @@
+    <header>
+      <nav>
+        <ul>
+          <li class="navName">Portfolio</li>
```
[ OPTIONAL ] Please wrap the logo in a link tag.

SajjadAhmad14 commented on index.html:
```diff
@@ -0,0 +1,107 @@
+
+        <div class="sideGradientright"></div>
+        <ul class="socials">
+          <li><i class="fab fa-github"></i></li>
+          <li><i class="fab fa-linkedin"></i></li>
+          <li><i class="fas fa-hand-peace"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li><i class="fab fa-facebook-square"></i></li>
```
[ OPTIONAL ] These icons should be clickable links.

SajjadAhmad14 reviewed (approved):
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Timo Wester, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
